src/cfnlint/custom_rules.py

```diff
@@ -0,0 +1,40 @@
+"""
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: MIT-0
+"""
+# pylint: disable=W0108
+# pylint: disable=W0622
+import logging
+import cfnlint
+import cfnlint.customRules.Operators
+import cfnlint.customRules.Rule
+
+LOGGER = logging.getLogger(__name__)
+Operator = {'EQUALS': lambda x, y: cfnlint.customRules.Operators.equalsOp(x, y),
+            'PLACEHOLDER': lambda x, y: LOGGER.debug('Placeholder Op')}
+
+def check(filename, template):
+    """ Process custom rule file """
+    matches = []
+    with open(filename) as customRules:
+        line_number = 1
+        for line in customRules:
+            LOGGER.debug('Processing Custom Rule Line %d', line_number)
+            line = line.replace('"', '')
+            rule = cfnlint.customRules.Rule.make_rule(line.split(' '))
+            if rule.valid and rule.resourceType[0] != '#':
+                try:
+                    template_properties = template.get_resource_properties([rule.resourceType])
```

Comment:
There is no such thing as template properties... did you mean `resource_properties`?

Reply:
Resource properties would be a much more appropriate name!

Reply:
Lol we need to rename `get_resource_properties` because it doesn't return the properties of one resource, it returns the properties of _all_ that match that type

Reply:
Yep into one big array! Maybe `get_resource_properties_array` or `get_resources_properties`